### ISSUE TITLE
Quit social media ... Cal Newport (author of deep work)

### ISSUE THREAD

**Issue by nelsonic** [nelsonic/nelsonic.github.io#273]
https://youtu.be/3E7hkPZ-HTk

[![image](https://cloud.githubusercontent.com/assets/194400/22543046/9fe242a2-e926-11e6-86b2-21dbc91d5834.png)](https://youtu.be/zfoCyFvADtU)
https://youtu.be/zfoCyFvADtU

**Comment by iteles** [comment]
I heard this podcast with Cal Newport earlier this year and immediately put his book in my amazon cart queue: http://www.asianefficiency.com/podcast/088-cal-newport/

**Comment by nelsonic** [comment]
Yeah, maybe _quitting_ "Social" (_Distraction!_) Media will be the catalyst that will allow us **Time** to focus on what matters...? ❓

**Comment by iteles** [comment]
https://twitter.com/iteles/status/813539229619851264

**Comment by nelsonic** [comment]
Yeah, aren't we _already_ in the second week of the new year...?
How did the _hiatus_ go...?

**Comment by iteles** [comment]
It wasn't long enough.

**Comment by nelsonic** [comment]
Does _quitting_ "Social Media" make me a "_Quitter_"...? 🤔  🤣 

![image](https://cloud.githubusercontent.com/assets/194400/21799683/32271880-d712-11e6-8c21-bc8ebe5a04b5.png)

**Comment by jedwards1211** [comment]
@nelsonic you're using GitHub quite like social media 😉

**Comment by nelsonic** [comment]
@jedwards1211 ***`Yes`*** I use GitHub to "_socially_" collaborate on code projects 
(and as a (_public_) "Todo List" / task tracker...)

The key distinction between my usage of GitHub and other "Social Media" sites is:
+ I `pay` GitHub for their service. 
+ I `GET` a lot of real _value_ out of using GitHub both for pleasure & work.
+ I `expect` GitHub aren't mining & selling my personal data to advertisers
+ Every feature on GitHub has an **API** which allows me to `extract` my data and _leave_ at any time.

Facebook, by contrast, are hoodwinking people into using their "_useful_" services (_[mostly systematically copied from other apps/sites](https://www.theguardian.com/technology/2016/dec/16/facebook-copies-snapchat-feature-for-15th-time) ...!_) while harvesting _all_ their personal data. 
I don't know if people care... 
Mostly people just want a "_free_" photo hosting service and to stay in contact with family/friends...

And Facebook (_and other "Social Media" services_) are "**Free**", right...?!
https://www.facebook.com/help/186556401394793
![facebook-is-free](https://cloud.githubusercontent.com/assets/194400/21828157/eb8223d2-d786-11e6-8232-4cbf9959068b.png)

Maybe not...

[![facebook-isnt-free](https://cloud.githubusercontent.com/assets/194400/21827970/dd899554-d785-11e6-8df9-1d1e718a1760.png)](https://www.youtube.com/watch?v=d3rS7I6Xyz8)

I would _gladly_ Pay **Mark Zuckerberg & Co**. ***$5 / month*** (_way more than he is selling my data for!!_) 
`if` there was a "***we won't ever sell your data***" `<option>` on Facebook. 
But that will _never_ happen because they have their Business Model now (_selling their user's personal information!!_) and the "Market" would not _allow_ them to change (_even if there was an internal appetite to "do the right thing"..._)

Meanwhile I have a `#plan` for quitting Facebook but I need _time_ to implement it!
+ Use Facebook API to extract all my contacts and birthdays so I can retain that _useful_ data (_so I can send people birthday greetings!_)
+ Use Facebook API to extract any other data they have on me.
+ Use messenger API to send PM to all my Fb contacts informing them ***Why*** I'm quitting.
+ Close/Delete Facebook account.
+ Get on with work.

Like your fine self, Andy, I've had the "[_come and work for us_](https://github.com/dwyl/learn-react/issues/23#issuecomment-271628776)" email from Fb recruiters several times. They obviously don't have a very good internal database for tracking the "**NO**s" or they are just "_persistent_"... 

P.S: I don't think Facebook is _necessarily_ "_worse_" than any other "Social Media"... 
They are all mining and selling personal data. This is just an _example_.

**Comment by jedwards1211** [comment]
Yeah, I agree that it would be nice to pay Facebook (etc.) in return for not selling my data.  Though I would also want the option to completely disable distracting features like the newsfeed, I have no idea if that's become possible since I quit 2 years ago...

Or that they had a way to subscribe to certain group notifications by e-mail only, without opening an account.  The only thing I really regret about not being on Facebook is sometimes missing out on group event notifications.

I'm glad to year you also turned their recruiters down!

**Comment by sirmews** [comment]
Any suggestions for dropping LinkedIn? While many professional devs can get away with using Github as their resume some of us have different occupations.

In terms of social media accounts or those that come close, I only have LinkedIn and Github.

**Comment by nelsonic** [comment]
@raonav I have a (_mental_) Plan to Quit LinkedIn too... 😜 
Historically it's been pretty _useful_ for me as my "CV".
I haven't "_applied_" for a "_Job_" in the past 9 years because people have _found_ me, 
often through LinkedIn... 
but there _is_ a better way, we just need to _build_ it...!

> I don't think MSFT would have paid $26Bn to acquire LNKD 
http://www.theverge.com/2016/12/6/13855408/microsoft-linkedin-acquisition-cleared 
to just let them "_keep doing what they are doing_" ... 
it's clear to me that MSFT are going to "_data mine_" the _crap_ out of the LinkedIn data
to _tighten_ their grip on the Enterprise Software market by knowing _everything_ about the "_Executives_" who make software purchasing decisions...

We _started_ making progress towards _eventually_ leaving LNKD in 2015 with an _internal_ project which I can tell you _more_ about if you're interested, but sadly, (_for now_) not _all_ of the code is "_Open Source_"...

**Comment by sirmews** [comment]
@nelsonic Pretty interested in hearing more about that project. I aim to drop LinkedIn this year. Their history of dark UX practices alone made me despise them, even before the MSFT purchase.

**Comment by nelsonic** [comment]
@raonav see: https://github.com/akitten and think of how these pieces can be _assembled_ ... 🙈

**Comment by nelsonic** [comment]
Good summary of Deep Work: https://youtu.be/LJ9lnspNNRE (1h)

**Comment by nelsonic** [comment]
[![image](https://cloud.githubusercontent.com/assets/194400/25478487/2f7fefaa-2b39-11e7-838f-4371d1111660.png)](https://youtu.be/d5GecYjy9-Q)
https://youtu.be/d5GecYjy9-Q

**Comment by nelsonic** [comment]
http://calnewport.com/blog/2018/02/09/facebooks-desperate-smoke-screen

**Comment by nelsonic** [comment]
https://youtu.be/HBRLMoL_vTQ
09:07 “instead of making more money the more time we get, 
we make more money the more we have _helped_ you in your life”

**Comment by nelsonic** [comment]
https://mashable.com/article/how-delete-facebook

**Comment by nelsonic** [comment]
![image](https://user-images.githubusercontent.com/194400/46594252-c3fa5400-cac9-11e8-8f7d-cd56267399a9.png)

https://ronaldlangeveld.com/deleting-facebook/
via/comments: https://news.ycombinator.com/item?id=18164188

**Comment by nelsonic** [comment]
https://www.nytimes.com/2018/10/10/technology/personaltech/how-to-delete-facebook-instagram-account.html
![delete-facebook-instagram](https://user-images.githubusercontent.com/194400/46937265-bc522680-d058-11e8-98b1-0863f1b82290.png)
comments: https://news.ycombinator.com/item?id=18215171

**Comment by iteles** [comment]
February 2019!
https://amzn.to/2St2Bmz

![image](https://user-images.githubusercontent.com/4185328/47665490-214d6680-db99-11e8-9ec7-ee7a922d7275.png)

**Comment by nelsonic** [comment]
@iteles looking forward to Cal's new book. 📆

**Comment by nelsonic** [comment]
20 Reasons To Quit Social Media: https://durmonski.com/life-advice/reasons-to-quit-social-media/
via: https://news.ycombinator.com/item?id=28616277 (but the discussion isn't very enlightening ...)

**Comment by nelsonic** [comment]
How do I permanently delete my Facebook account? https://www.facebook.com/help/224562897555674/
via: https://news.ycombinator.com/item?id=28770937
